src/status_im/utils/ethereum/stickers.cljs

```diff
@@ -18,14 +18,14 @@
   (ethereum/call web3
                  (ethereum/call-params contract "getPackData(uint256)" (ethereum/int->hex pack-id))
                  (fn [_ data]
-                   (cb (abi-spec/decode (subs data 2) ["bytes4[]" "address" "bool" "uint256" "uint256" "bytes"])))))
+                   (cb (when-not (empty? (subs data 2)) (abi-spec/decode (subs data 2) ["bytes4[]" "address" "bool" "uint256" "uint256" "bytes"]))))))
```

Comment:
is it intended that you call the cb function without arguments if the data is empty?
can you put (subs data 2) in a let binding and give it a name?

Reply:
yes it's intended